eq-author/src/App/page/Design/CalculatedSummaryPageEditor/AnswerSelector.js

```diff
@@ -175,13 +180,33 @@ export class UnwrappedAnswerSelector extends Component {
   };
 
   renderAnswers(answers, answerType) {
+
+    // console.log('not empty state props', this.props);
+
     const { section } = this.props.page;
-    const errorValidationMsg = this.props.getValidationError({
+
+    const unitInconsistencyError = this.props.getValidationError({
       field: "summaryAnswers",
       message: CALCSUM_SUMMARY_ANSWERS_THE_SAME,
     });
 
-    const isInvalid = Boolean(errorValidationMsg);
+    const minOfTwoAnswersError = this.props.getValidationError({
+      field: "summaryAnswers",
+      message: CALCSUM_ANSWER_NOT_SELECTED,
+    });
+
+    const unit = this.props.page.summaryAnswers[0].properties.unit;
+
+    let errorMsg;
+
+    if (unit !== undefined) {
+      errorMsg = buildLabelError(CALCSUM_ANSWER_NOT_SELECTED, `${unit}`, 20, 19);
+    } else {
+    errorMsg = CALCSUM_ANSWER_NOT_SELECTED;
+    }
+
+    const isInvalid = Boolean(unitInconsistencyError) || Boolean(minOfTwoAnswersError);
```

Comment:
I don't think the explicit casting to `Boolean` is needed (I don't think it was really needed before either). I tried it out locally and the `ErrorContext` styling still works just fine with implicit casting.

```suggestion
    const isInvalid = unitInconsistencyError || minOfTwoAnswersError;
```

Reply:
agreed - thanks Tom